electrolytes/solvation_shell_extract.py

```diff
@@ -0,0 +1,181 @@
+import os
+import logging
+import argparse
+import numpy as np
+from tqdm import tqdm
+from collections import defaultdict
+import matplotlib.pyplot as plt
+import MDAnalysis as mda
+import nglview as nv
+from solvation_analysis.solute import Solute
+from solvation_analysis._column_names import *
+from pymatgen.core.structure import Molecule
+from solvation_shell_utils import filter_by_rmsd
+
+logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"))
+
+
+def extract_solvation_shells(
```

Comment:
In terms of coding best practice, this function should probably be broken up into sub-functions.